x-pack/libbeat/management/managerV2.go

```diff
@@ -332,7 +332,11 @@ func (cm *BeatV2Manager) handleUnitReload(unit *client.Unit) {
 // Handle the updated config for an output unit
 func (cm *BeatV2Manager) handleOutputReload(unit *client.Unit) {
 	_, _, rawConfig := unit.Expected()
-	cm.logger.Debugf("Got Output unit config %s", rawConfig.GetId())
+	if rawConfig == nil {
+		cm.logger.Warnf("got output update with no config, ignoring")
```

Comment:
[Suggestion]

I'd log that as info because it looks like just an information about the config received rather than a potential error.